src/docs/ganache/quickstart.md

```diff
@@ -10,12 +10,12 @@ This quickstart guide will walk you through installing Ganache and creating a pe
 If this isn't your first time using Ganache, or you already know you'll need custom configuration options, check out the [Creating Workspaces documentation](/docs/ganache/workspaces/creating-workspaces).
 
 <p class="alert alert-info">
-<i class="far fa-info-circle"></i> <strong>Working with Ethereum and prefer using the command line?</strong>: This page will focus only on the graphical interface. Please see the <a href="https://github.com/trufflesuite/ganache-cli/blob/master/README.md">Ganache CLI Readme</a> for more information on the command line flavor of Ganache.
+<i class="far fa-info-circle"></i> <strong>Working with Ethereum and prefer using the command line?</strong>: This page will focus only on the graphical interface. Please see the <a href="https://github.com/trufflesuite/ganache-cli-archive/blob/master/README.md">Ganache CLI Readme</a> for more information on the command line flavor of Ganache.
 </p>
 
 ## 1. Install Ganache
 
-[Download](https://github.com/trufflesuite/ganache/releases) the appropriate version for your OS:
+[Download](https://github.com/trufflesuite/ganache-ui/releases) the appropriate version for your OS:
```

Comment:
Maybe there is a better destination for this?

Reply:
Is there a way to link to the latest release directly?